pkg/config/system_probe.go

```diff
@@ -294,9 +294,15 @@ func InitSystemProbeConfig(cfg Config) {
 	cfg.BindEnvAndSetDefault("runtime_security_config.activity_dump.max_dump_count_per_workload", 25)
 	cfg.BindEnvAndSetDefault("runtime_security_config.activity_dump.tag_rules.enabled", false)
 
-	// SBOM
+	// CWS - SBOM
 	cfg.BindEnvAndSetDefault("runtime_security_config.sbom.enabled", false)
 	cfg.BindEnvAndSetDefault("runtime_security_config.sbom.workloads_cache_size", 10)
+
+	// CWS - Security Profiles
+	cfg.BindEnvAndSetDefault("runtime_security_config.security_profile.enabled", false)
```

Comment:
Not a huge fan of this, I think this will cause confusion because activity dumps are kind of already called security profiles in the frontend.

Reply:
Happy to change to whatever you think might be better ! (you meant security snapshots no ?)